packages/api/tests/endpoints/test-rules.js

```diff
@@ -1814,8 +1820,8 @@ test.serial('PATCH keeps initial trigger information if writing to PostgreSQL fa
     pgCollection,
   } = t.context;
 
-  const topic1 = await awsServices.sns().createTopic({ Name: randomId('topic1_') });
-  const topic2 = await awsServices.sns().createTopic({ Name: randomId('topic2_') });
+  const topic1 = await await awsServices.sns().send(new CreateTopicCommand({ Name: randomId('topic1_') }));
```

Comment:
Double `await`s here and a few places below this line.

Reply:
🤦 Well. This is embarrassing. 

https://github.com/nasa/cumulus/pull/3606/commits/ff84c81f44548ffd29b0f653d67c858af46ae1e1